Persistence/Repos/IBank.cs

```diff
@@ -159,19 +159,14 @@ public async Task<int> GetReservedMoney(T user)
 
         public abstract Task<int> GetTotalMoney(T user);
 
-        public async Task<int> GetAvailableMoney(T user)
-        {
-            return await GetTotalMoney(user) - await GetReservedMoney(user);
-        }
+        public async Task<int> GetAvailableMoney(T user) =>
+            await GetTotalMoney(user) - await GetReservedMoney(user);
 
         public abstract Task<IList<TransactionLog>> PerformTransactions(
```

Comment:
not particularly a fan of this formatting, but I guess it's ok

Reply:
😮

in what way?

Reply:
it would probably look better on one line, I don't think it's too long for that?